pandas/core/groupby/groupby.py

```diff
@@ -1089,6 +1088,12 @@ def mean(self, *args, **kwargs):
         -------
         pandas.Series or pandas.DataFrame
 
+        See Also
+        --------
+        pandas.Series.%(name)s
+        pandas.DataFrame.%(name)s
+        pandas.Panel.%(name)s
```

Comment:
To avoid repeating this all the time, you can add here `%(see_also)s`, and decorate the function with `@Substitution(see_also=_doc_template)`.

And if you can change the name `_doc_template` to something more descriptive of what it is, like `_common_see_also`, that would be really great.

Reply:
Sure thing, see cabe06200f55a7c0b9fbfa7a36ae68605be7121a

Reply:
were those changes undone? I can see them

Reply:
Yeah I did as I experienced some issues with what I had changed. See 14fe7296f5c973b07e5414545c5b2a1479aafaad